projects/templates/plop-templates/template.ts.hbs

```diff
@@ -0,0 +1,13 @@
+import { html, TemplateResult } from '@spectrum-web-components/base';
+
+import '@spectrum-web-components/{{ name }}/{{> tagnamePartial }}.js';
+
+export interface Properties {
+}
+
+export const Template = (props: Properties): TemplateResult => {
+    // Remove the console.log() when you're ready to consume `props` in the element below.
+    // eslint-disable-next-line no-console
+    console.log('template properties', props);
```

Comment:
I'm not a big fan of leaving code in a template that _should_ be removed, but I couldn't think of a better way to include `props` in the `Template` without running into TS errors otherwise.